### ISSUE TITLE
HTTP 404 on `/_vercel/insights/script.js`

### ISSUE THREAD

**Issue by agurtovoy** [vercel/analytics#49]
Firstly, thank you for offering built-in analytics support, this by itself makes it worth upgrading to Pro for me!

I followed the steps in the [Audiences Quickstart Guide](https://vercel.com/docs/concepts/analytics/audiences/quickstart), including redeployment, but I'm still getting 404 on a GET for `/_vercel/insights/script.js`:

<img width="802" src="https://user-images.githubusercontent.com/3394202/219712554-7dc91878-8b01-417c-9639-af936176f3a7.png">

... and, consequently, my production dashboard shows 0 visitors:

<img width="1239" src="https://user-images.githubusercontent.com/3394202/219713369-3a7dceb3-5083-4e22-a20f-ee12d34abc79.png">

Interestingly enough, my preview deployments don't have this issue and are actually able to track visitors, but I think it's because they are doing it through a slightly different mechanism:

<img width="804" src="https://user-images.githubusercontent.com/3394202/219713814-21590245-fcba-4935-8f94-2c9da06ca196.png">

<img width="1248" src="https://user-images.githubusercontent.com/3394202/219713988-68887c80-3d8c-45df-ba7c-f69bb62d5aa1.png">

I saw that https://github.com/vercel/analytics/issues/1 was confirmed as resolved by people who were experiencing it earlier, so thought I'd open a new issue.

**Comment by tobiaslins** [comment]
Hey @agurtovoy!
I just looked into this, it seems like you did deploy to production before analytics got enabled. Can you please try redeploying production and check if its working afterward.

Thanks :)

**Comment by agurtovoy** [comment]
Hey @tobiaslins, thanks so much for the quick response! 

Redeployment did fix this, and I think I know what I did wrong: I redeployed after enabling **Vercel Analytics**, which is the prompt that is shown by default when you click on a project's Analytics tab, but I guess one also need to redeploy after enabling **Audiences**. 

The docs say as much, but, unlike the Vercel Analytics dialog, there is no explicit "Redeploy" button when you enable Audiences, so if you add this package first and then enable Audiences, you'll get what I got:

| Vercel Analytics prompt | Audiences prompt |
| ------------- | ------------- |
|  <img width="485" src="https://user-images.githubusercontent.com/3394202/219763468-46b962e7-56d1-4a78-95cb-08a86a8c0fcf.png"> | <img width="534" src="https://user-images.githubusercontent.com/3394202/219763692-def8e5f8-efce-457d-91ac-7a7fb94531ab.png"> |

It's definitely a user error, but perhaps there is something you could do with the UX to make this more clear.

Thanks again for the prompt reply!

**Comment by tobiaslins** [comment]
Thank you for the feedback! We're currently improving the setup flow because it can be confusing at the moment!

**Comment by 24jr** [comment]
<img width="794" alt="Screen Shot 2023-03-13 at 12 57 53 AM" src="https://user-images.githubusercontent.com/34662182/224611784-efe74ce3-05d0-40cd-8a38-f44b7454eca3.png">
<img width="445" alt="Screen Shot 2023-03-13 at 12 59 24 AM" src="https://user-images.githubusercontent.com/34662182/224611919-cbd10dc6-2532-4c16-8953-0f398268ac3e.png">


Im confused, so what is the solution? I've tried disabling, reenabling analytics, redeploying, etc and still just looks like this and on actual website see error. I installed npm i @vercel/analytics into my sveltekit app and deployed and enabled. 

This project may be taken down by time someone sees it but this https://judebot.vercel.app/ does it not work on dev site?

Also added 
```
<script>
  import { dev } from "$app/environment"
  import { inject } from "@vercel/analytics"

  inject({ mode: dev ? "development" : "production" })
</script>
``` 
to my +layout.svelte file (not that i have an env var named env but maybe that doesn't matter
